pkg/manager/internal.go

```diff
@@ -538,3 +543,7 @@ func (cm *controllerManager) startLeaderElection() (err error) {
 	go l.Run(ctx)
 	return nil
 }
+
+func (cm *controllerManager) Elected() <-chan struct{} {
```

Comment:
`IsLeader` or `Leading` would probably be better names, the election is the process, leading is the state we care about. Also, please add godocs.

Reply:
`Leading` seems like an appropriate name, considering we may become "leader" without holding an election.